### PR TITLE
chore/dont-enforce-non-empty-newlines-between-imports

### PR DESCRIPTION
**Pull request by mthomasuk**
do NOT force people to group all of their imports

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ module.exports = {
       "error",
       {
         "groups": ["builtin", "external", "internal"],
-        "newlines-between": "always",
+        "newlines-between": "ignore",
```
Comment (alecdotico) [inline index.js:58]: why not? can you develop pros&cons?

Comment (mthomasuk) [reply index.js:58]: Screenshot 2020-12-11 at 11 05 13

Comment (mthomasuk) [reply index.js:58]: tl;dr: eslint doesn't know the difference between `~/utils` and `~/components` as separate import groups

Comment (alecdotico) [reply index.js:58]: Import-wise, they are the same kind of internal import. But sure, I'm not opinionated on new lines or the lack of it

**Review by alecdotico — APPROVED** [review]
🥘

**Pull request merged**